packages/storefront/src/components/PartialDocs.vue

```diff
@@ -77,7 +85,35 @@ export default (targetOptions: TargetOptions, indexHtml: string): string => {
   ${angularPartials}
 
   return indexHtml.replace(/<\\/${this.location}>/, \`\${partialContent}$&\`);
-};`;
+};
+
+yarn add --dev glob
+
+<!-- karma.conf -->
```

Comment:
Why not put this on a dedicated page in the onboarding similar to other frameworks? https://designsystem.porsche.com/housekeeping/fix-components-3.7.0-rc.1/developing/react/testing
Currently this is just duplicated a bunch of times in each partial demo without any context and little value.